<ui version="4.0" >
 <class>ImageViewer</class>
 <widget class="QMainWindow" name="ImageViewer" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Qt Mini Image Viewer</string>
  </property>
  <property name="statusTip" >
   <string>Histopatho-GTM (GroudTruthMaker) by Chao-Hui Huang (Ph.D.) @ Image &amp; Pervasive Access Lab (IPAL), French-Singaporean International Joint Laboratory (CNRS UMI 2955, A*STAR/I2R, NUS, UJF)</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QHBoxLayout" >
    <property name="spacing" >
     <number>6</number>
    </property>
    <property name="margin" >
     <number>9</number>
    </property>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QDockWidget" name="dockWidget" >
   <property name="features" >
    <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas" >
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea" >
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents" >
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>9</number>
     </property>
     <item>
      <widget class="QTreeView" name="treeView" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="autoFillBackground" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="toolButtonStyle" >
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad" />
   <addaction name="actionSave" />
   <addaction name="actionPrevious" />
   <addaction name="actionNext" />
   <addaction name="actionZoomOut" />
   <addaction name="actionNormalSize" />
   <addaction name="actionZoomIn" />
   <addaction name="actionAbout" />
  </widget>
  <widget class="QToolBar" name="NuclearPleomorphismExamBar" >
   <property name="windowTitle" >
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionNuclearPleomorphism_N" />
   <addaction name="actionNuclearPleomorphism_1" />
   <addaction name="actionNuclearPleomorphism_2" />
   <addaction name="actionNuclearPleomorphism_3" />
  </widget>
  <widget class="QToolBar" name="TubuleFormationExamBar" >
   <property name="windowTitle" >
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionTubuleFormation_N" />
   <addaction name="actionTubuleFormation_1" />
   <addaction name="actionTubuleFormation_2" />
   <addaction name="actionTubuleFormation_3" />
  </widget>
  <widget class="QToolBar" name="MitoticCountExamBar" >
   <property name="windowTitle" >
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionMitoticCount_N" />
   <addaction name="actionMitoticCount_1" />
   <addaction name="actionMitoticCount_2" />
   <addaction name="actionMitoticCount_3" />
  </widget>
  <action name="actionAbout" >
   <property name="icon" >
    <iconset resource="Histopatho-GTMRes.qrc" >
     <normaloff>:/images/about.png</normaloff>:/images/about.png</iconset>
   </property>
   <property name="text" >
    <string>About</string>
   </property>
   <property name="statusTip" >
    <string>Show program about dialog</string>
   </property>
   <property name="shortcut" >
    <string>F1</string>
   </property>
  </action>
  <action name="actionNext" >
   <property name="icon" >
    <iconset resource="Histopatho-GTMRes.qrc" >
     <normaloff>:/images/next.png</normaloff>:/images/next.png</iconset>
   </property>
   <property name="text" >
    <string>Next</string>
   </property>
   <property name="shortcut" >
    <string>Space</string>
   </property>
  </action>
  <action name="actionPrevious" >
   <property name="icon" >
    <iconset resource="Histopatho-GTMRes.qrc" >
     <normaloff>:/images/previous.png</normaloff>:/images/previous.png</iconset>
   </property>
   <property name="text" >
    <string>Previous</string>
   </property>
   <property name="shortcut" >
    <string>Backspace</string>
   </property>
  </action>
  <action name="actionZoomIn" >
   <property name="icon" >
    <iconset resource="Histopatho-GTMRes.qrc" >
     <normaloff>:/images/zoomin.png</normaloff>:/images/zoomin.png</iconset>
   </property>
   <property name="text" >
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoomOut" >
   <property name="icon" >
    <iconset resource="Histopatho-GTMRes.qrc" >
     <normaloff>:/images/zoomout.png</normaloff>:/images/zoomout.png</iconset>
   </property>
   <property name="text" >
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionNormalSize" >
   <property name="icon" >
    <iconset resource="Histopatho-GTMRes.qrc" >
     <normaloff>:/images/zoomnormal.png</normaloff>:/images/zoomnormal.png</iconset>
   </property>
   <property name="text" >
    <string>Normal Size</string>
   </property>
  </action>
  <action name="actionNuclearPleomorphism_N" >
   <property name="text" >
    <string>(&amp;Q) Nuclear Pleomorphism Score: N/A</string>
   </property>
  </action>
  <action name="actionNuclearPleomorphism_1" >
   <property name="text" >
    <string>(&amp;W) Nuclear Pleomorphism Score: 1</string>
   </property>
  </action>
  <action name="actionNuclearPleomorphism_2" >
   <property name="text" >
    <string>(&amp;E) Nuclear Pleomorphism Score: 2</string>
   </property>
  </action>
  <action name="actionNuclearPleomorphism_3" >
   <property name="text" >
    <string>(&amp;R) Nuclear Pleomorphism Score: 3</string>
   </property>
  </action>
  <action name="actionTubuleFormation_N" >
   <property name="text" >
    <string>(&amp;A) Tubule Formation Score: N/A</string>
   </property>
  </action>
  <action name="actionTubuleFormation_1" >
   <property name="text" >
    <string>(&amp;S) Tubule Formation Score: 1</string>
   </property>
  </action>
  <action name="actionTubuleFormation_2" >
   <property name="text" >
    <string>(&amp;D) Tubule Formation Score: 2</string>
   </property>
  </action>
  <action name="actionTubuleFormation_3" >
   <property name="text" >
    <string>(&amp;F) Tubule Formation Score: 3</string>
   </property>
  </action>
  <action name="actionMitoticCount_N" >
   <property name="text" >
    <string>(&amp;Z) Mitotic Count Score: N/A</string>
   </property>
  </action>
  <action name="actionMitoticCount_1" >
   <property name="text" >
    <string>(&amp;X) Mitotic Count Score: 1</string>
   </property>
  </action>
  <action name="actionMitoticCount_2" >
   <property name="text" >
    <string>(&amp;C) Mitotic Count Score: 2</string>
   </property>
  </action>
  <action name="actionMitoticCount_3" >
   <property name="text" >
    <string>(&amp;V) Mitotic Count Score: 3</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="icon" >
    <iconset>
     <normaloff>://images/save.png</normaloff>://images/save.png</iconset>
   </property>
   <property name="text" >
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad" >
   <property name="icon" >
    <iconset>
     <normaloff>://images/open.png</normaloff>://images/open.png</iconset>
   </property>
   <property name="text" >
    <string>Load</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Histopatho-GTMRes.qrc" />
 </resources>
 <connections/>
</ui>
